server/quota/quota.go

```diff
@@ -168,7 +227,19 @@ func (m *Manager) updateTenantMetrics(ctx context.Context, namespace string) {
 
 		s.TenantSizeUpdateAt.Store(currentTimeStamp)
 		metrics.UpdateNameSpaceSizeMetrics(namespace, sz)
-		m.updateTenantSize(ctx, namespace)
+		err := m.updateTenantSize(ctx, namespace)
```

Comment:
Do not return error from updateTenantSize, so as there is no action on it and the error has already been logged in getTenantSize

Reply:
Makes sense, I changed this.